exercise1/assets/functions.js

```diff
@@ -0,0 +1,108 @@
+'use strict'
+
+window.onload = function(){
+
+	setTimeout(function(){ 
+	document.getElementById('hide').style.opacity = "1";}, 900)
+}
+
+/* function clickme(){
+
+	alert("You click me!");
+
+} */
+
+
+/* exercise 2*/
+function sendAjax (config){
+	var url = config.url;
+	var method = config.type;
+	
+	var joke = new Promise(function(done, reject){
+
+		var ajax = new XMLHttpRequest();
+		ajax.open( method , url, true);
```

Comment:
Why do you leave spaces in between the `(` and the first parameter if not leaved in all the method calls?